src/views/router_views/stopScreen/problem-with-psc-data.njk

```diff
@@ -0,0 +1,16 @@
+{% set title = i18n.problem_with_psc_data_main_title %}
+
+{% extends "layouts/default.njk" %}
+
+{% block main_content %}
+
+<h1 class="govuk-heading-l">{{ i18n.problem_with_psc_data_main_title }}</h1>
+<p class="govuk-body">
+  {{ i18n.problem_with_psc_data_body }}
+</p>
+<p class="govuk-body">
+  <a class="govuk-link" href="mailto: enquiries@companieshouse.gov.uk">{{ i18n.problem_with_psc_data_email }}</a>
```

Comment:
ought the email address be passed to the template, like it is done for the Super Secure screen?

Reply:
done